modules/waypoint.py

```diff
@@ -0,0 +1,17 @@
+"""
+Class Waypoint to use instead of tuple for coordinates.
+""" 
+
+class Waypoint:
+    def __init__(self, name: str, latitude: float, longitude: float):
```

Comment:
Add constructor docstring.

Explain latitude and longitude are in decimal degrees.